helix-core/src/main/java/org/apache/helix/task/GenericTaskAssignmentCalculator.java

```diff
@@ -65,9 +65,9 @@
 
   @Override
   public Map<String, SortedSet<Integer>> getTaskAssignment(CurrentStateOutput currStateOutput,
-      ResourceAssignment prevAssignment, Collection<String> instances, JobConfig jobCfg,
-      final JobContext jobContext, WorkflowConfig workflowCfg, WorkflowContext workflowCtx,
-      Set<Integer> partitionSet, Map<String, IdealState> idealStateMap) {
+      Collection<String> instances, JobConfig jobCfg, final JobContext jobContext,
```

Comment:
Better not change public API, we can deprecate it and add a new one.

Reply:
Done.